### PR TITLE
Upgrade to spring-boot2

### PR DESCRIPTION
**Pull request by harsimranmaan**

### CODE REVIEW

**Comment by harsimranmaan** [comment]
@clun Do you mind checking the build? Looks like a java8 install issue

File: pom.xml [hunk 1]
```diff
@@ -129,10 +130,10 @@
         <version.maven.plugin.jar>2.6</version.maven.plugin.jar>
         <version.maven.plugin.dependency>2.8</version.maven.plugin.dependency>
 
-        <spring-boot.version>1.3.2.RELEASE</spring-boot.version>
+        <spring-boot.version>2.1.6.RELEASE</spring-boot.version>
```
Comment (paul58914080) [inline pom.xml]: I think the right thing would be to remove this dependency as this has been moved to https://github.com/ff4j/ff4j-spring-boot-starter-parent

Comment (clun) [reply pom.xml]: yeah totally, other updrates are related to thymeleaf v3 and make sens, will probable merge and then remove the spring-boot dependency yeah.

Comment (clun) [reply pom.xml]: + Spring boot 2.x expect java 11 where the ff4j v1 should stay compliant with jdk8.

**Pull request merged**